content/en/real_user_monitoring/guide/rum-for-product-analytics.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use RUM & Session Replay for Product Analytics
+kind: guide
+description: Learn how to use RUM & Session Replay to monitor trends in user behavior and feature adoption.
+further_reading:
+- link: '/real_user_monitoring/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn how to visualize your RUM data'
+- link: '/real_user_monitoring/frustration_signals'
+  tag: 'Documentation'
+  text: 'Learn about frustration signals'
+- link: '/real_user_monitoring/session_replay'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+- link: '/dashboards/guide/powerpacks-best-practices/'
+  tag: 'Documentation'
+  text: 'Learn about powerpacks'
+---
+
+## Overview
+
+[RUM & Session Replay][1] allows you to monitor trends in consumer behavior and uncover answers to product usage questions about your web and mobile applications.
+
+This guide discusses several use cases to enrich your RUM & Session Replay data and provides example queries you can use to answer questions about product analytics.
+
+## Setup
+
+Once you have set up the Datadog RUM SDK, enrich your [browser][2] or mobile ([iOS][3] and [Android][4]) data with attributes to customize the data according to your use case. For example, adding contextual information allows you to [identify sessions that are tied to specific users][4].
+
+## Monitor page traffic and button usage
+
+If you are interested in learning what buttons your users click on the most, you can track page traffic and the usage of buttons in your application. 
+
+1. Navigate to the [RUM Explorer][5] and select **Actions** from the dropdown menu next to the search query. 
+2. Enter `@view.name:/cart` in the search query and select the **Top List** visualization type.
+3. In the `by` field of the `Group into fields` section above, select **Action Name** from the group dropdown.
+
+This example displays the top actions on Shopist's `/cart` page.
+
+{{< img src="real_user_monitoring/guide/rum-for-product-analytics/actions_in_cart_page.png" alt="Search query for actions on Shopist's Cart page" style="width:90%;">}}
+
+For example: 
+
+{{< code-block lang="json" filename="json" disable_copy="false" collapsible="true" >}}
```

Comment:
you were going to add something here to introduce the JSON snippet right? can we please add a line before this one to say something like "if you want to import the JSON snippet to a dashboard, see the following"